src/control/server/harness.go

```diff
@@ -173,7 +173,7 @@ func (h *IOServerHarness) AwaitStorageReady(ctx context.Context, skipMissingSupe
 				continue
 			}
 		}
-		h.log.Info("SCM format required")
+		h.log.Infof("SCM format required on instance %d", instance.Index)
```

Comment:
instance.Index is a getter function, not a struct member. This will print the address of the function.